src/commands/importExportEntities/entities/languages.ts

```diff
@@ -1,11 +1,150 @@
+        })
+        .filter(notNull)
+      )
+    }
+  },
 };
+
+const createReplaceCodenameOperation = (codename: string): LanguageModels.IModifyLanguageData => ({
+  'op': 'replace',
+  'property_name': 'codename',
+  'value': codename
+})
+
+const createReplaceNameOperation = (name: string): LanguageModels.IModifyLanguageData => ({
+  'op': 'replace',
+  'property_name': 'name',
+  'value': name
+})
+
+const createReplaceFallbackLanguageOperation = (codename: string): LanguageModels.IModifyLanguageData => ({
+  'op': 'replace',
+  'property_name': 'fallback_language',
+  'value': { codename }
+})
+
+const createReplaceIsActiveOperation = (isActive: boolean): LanguageModels.IModifyLanguageData => ({
+  'op': 'replace',
+  'property_name': 'is_active',
+  'value': isActive
+})
+
+
+const updateProjectLanguage = async (client: ManagementClient, projectLanguage: LanguageModels.LanguageModel, importLanguage: LanguageContracts.ILanguageModelContract) => {
+  const operations: LanguageModels.IModifyLanguageData[] = [];
+
+  if (projectLanguage.codename !== importLanguage.codename) {
+    operations.push(createReplaceCodenameOperation(importLanguage.codename));
+  }
+
+  if (projectLanguage.name !== importLanguage.name) {
+    operations.push(createReplaceNameOperation(importLanguage.name));
+  }
+
+  if (projectLanguage.id !== defaultLanguageId && !projectLanguage.isActive) {
+    operations.push(createReplaceIsActiveOperation(true));
+  }
+
+  if (operations.length > 0) {
+    return client
+      .modifyLanguage()
+      .byLanguageCodename(projectLanguage.codename)
+      .withData(operations)
+      .toPromise();
+  }
+
+  return Promise.resolve();
+}
+
+const importLanguagesToProject = async (client: ManagementClient, importLanguages: ReadonlyArray<LanguageContracts.ILanguageModelContract>) => {
+  const projectLanguages = await client.listLanguages().toAllPromise().then(res => res.data.items);
+
+  await serially(importLanguages
+    .filter(l => l.id !== defaultLanguageId)
+    .map(importLanguage => () => {
+      const projectLanguage = projectLanguages.find(l => l.codename === importLanguage.codename);
+
+      const otherProjectLanguageMatchOnName = projectLanguages.find(l => l.name === importLanguage.name && l.codename !== importLanguage.codename);
+
+      if (otherProjectLanguageMatchOnName) {
+        throw new Error(`Could not update name of the language with codename ${importLanguage.codename}. The other language with codename ${otherProjectLanguageMatchOnName.codename} has the same name. Fix your languages names to continue`);
+      }
+
+      if (projectLanguage) {
+        return updateProjectLanguage(client, projectLanguage, importLanguage)
+      }
+
+      return client
+        .addLanguage()
+        .withData({
+          name: importLanguage.name,
+          codename: importLanguage.codename,
+          is_active: true,
+          external_id: importLanguage.external_id ?? `${importLanguage.codename}`.slice(0, 50),
```

Comment:
Why do you put codename into the template string, codename is a string on its own. 😄